### ISSUE TITLE
LoadError: cannot load such file -- algolia/insights

### ISSUE THREAD

**Issue by mkiser** [algolia/jekyll-algolia#108]
Seems like something changed with `algoliasearch` because Jekyll build now fails on `Jekyll-Algolia` as of a few hours ago 

Error => https://travis-ci.org/mkiser/WTFJHT/builds/470107663#L645

Here's my repo: https://github.com/mkiser/WTFJHT

I cleared the Travis-CI cache, too, but it still errors. Commenting out `Jekyll-Algolia` resolves the issue. 

```
$ ruby --version
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
$ rvm --version
rvm 1.29.3 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
$ bundle --version
Bundler version 1.17.2
$ gem --version
3.0.0
```

**Comment by pixelastic** [comment]
Thanks. Seems like we failed to update correctly the dependencies of `algoliasearch`. You can add `gem 'algoliasearch', '1.24'` to your Gemfile temporarily to force using the latest working version.

**Comment by julienbourdeau** [comment]
@mkiser Thank you for reporting this issue.

I released 1.25.0 earlier today but didn't register everything correctly in the `gemspec`. @pixelastic helped me fixing it and 1.25.1 is out, everything should be fin now.

Thanks again 👍

**Comment by pixelastic** [comment]
Thanks @mkiser, we're on it. Seems like we have a few hiccups with the release of the ruby client. We're working on it, to make sure all needed files are correctly sent to rubygems.

In the meantime, I released v1.4.10 of `jekyll-algolia` that pins the dependency to `1.24` (last working version), so the error should now disappear from your CI quickly.

**Comment by pixelastic** [comment]
And... issue is fixed in `algoliasearch` 1.25.2, so I released `jekyll-algolia` 1.4.11 that uses it :)

Everything should be all green now
